frontend/src/components/MealItem.js

```diff
@@ -2,14 +2,13 @@ const MealItem = (props) => {
     return (
         <div className="row mealItem">
             <div className="col">{props.name}</div>
-            <div className="col">{props.allergens}</div>
+            <div className="col allergens">{props.allergens.map(function(allergen) {
+                return (
+                    <span className={"allergen-" + allergen.toLowerCase()}>•</span>
+                );
+            })}</div>
```

Comment:
This can be written with an arrow function (part of ES6 syntax)
```
<div className="col allergens">
    {props.allergens.map((allergen) => <span className={"allergen-" + allergen.toLowerCase()}>•</span>)}
</div>
```